templates/settings.py.erb

```diff
@@ -23,6 +23,10 @@ REDIS_URL = "redis://localhost:<%= scope['redis::port'] %>/<%= scope['pulpcore::
 
 USE_NEW_WORKER_TYPE = <%= scope['pulpcore::use_rq_tasking_system'] ? "False" : "True" %>
 
+<% if scope['pulpcore::worker_ttl'] -%>
+WORKER_TTL = <%= scope['pulpcore::worker_ttl'] %>
+<% end -%>
```

Comment:
```suggestion
WORKER_TTL = <%= scope['pulpcore::worker_ttl'] %>
```

[Keeping with my suggestion above to make it explicit... also I realize we'd need to make the parameter not `Optional` in that case. I'll update the other comment with that]

NIT: I also suggested removing the empty line above this, so that it is in a common block with `USE_NEW_WORKER_TYPE`. It's easier for everyone to read (including ourselves) when settings related to a common feature are grouped together.